internal/resource/common/policyaction/policy_action_schema.go

```diff
@@ -81,7 +81,7 @@ func commonAttributeRulesAttr() schema.Attribute {
 
 func apcMappingPolicyActionSchema() schema.SingleNestedAttribute {
 	attrs := commonPolicyActionSchema()
-	attrs["attribute_mapping"] = attributemapping.ToSchema()
+	attrs["attribute_mapping"] = attributemapping.ToSchema(false)
```

Comment:
This one is required per the API doc page

Reply:
whoops, nice catch

Reply:
these are updated